### PR TITLE
fix grammar errors and update docs to use singular they -Do Not Merge Yet

### PR DESCRIPTION
**Pull request by ghost**
I came across some places "an" was used incorrectly, as well as some typos.  There were also some awkward usages of 'his' rather than 'their', 'his or her / he or she' rather than their/they.

### CODE REVIEW

**Comment by mypil** [comment]
@nenonix @JohanFreelancer9 - kindly do the copy editor review for this PR. Thanks.

File: microsoft-365/compliance/data-spillage-scenariosearch-and-purge.md
Comment (JohanFreelancer9) [inline microsoft-365/compliance/data-spillage-scenariosearch-and-purge.md]:
```suggestion
- To control which user mailboxes a data spillage investigator can search and manage and who can access the case, you can set up compliance boundaries and create a custom role group, which is described in [Step 1](#optional-step-1-manage-who-can-access-the-case-and-set-compliance-boundaries). To do this, you have to be a member of the Organization Management role group or be assigned the role management role. If you or in administrator in your organization has already set compliance boundaries, you can skip Step 1.
```

File: microsoft-365/compliance/eu-social-security-number-or-equivalent-id.md
Comment (JohanFreelancer9) [inline microsoft-365/compliance/eu-social-security-number-or-equivalent-id.md]:
```suggestion
  - Up to 1990, the assignment of a serial number corresponded to the county where the bearer of the number was born or (if born before 1947) where they had been living, according to tax records, on January 1, 1947, with a special code (usually 9 as the 7th digit) for immigrants 
```

File: microsoft-365/compliance/gdpr-arc-Azure.md
Comment (JohanFreelancer9) [inline microsoft-365/compliance/gdpr-arc-Azure.md]:
```suggestion
| ***Organization of Information Security Customer consideration(6.3)*** | The customer should, within their organization, define responsibilities for security and protection of personal data. This may include establishing specific roles to oversee privacy-related matters, including a DPO. Appropriate training and management support should be provided to support these roles. | An overview of the role of Microsoft's Data Protection Officer, the nature of their duties, reporting structure, and contact information.<br>- Microsoft DPO Information [[17](gdpr-arc-Azure.md#17)] | (37)(1)(a), (37)(1)(b), (37)(1)(c), (37)(2), (37)(3), (37)(4), (37)(5), (37)(6), (37)(7), (38)(1), (38)(2), (38)(3), (38)(4), (38)(5), (38)(6), (39)(1)(a), (39)(1)(b), (39)(1)(c), (39)(1)(d), (39)(1)(e), (39)(2) |
```

File: microsoft-365/compliance/gdpr-arc-Azure.md
Comment (JohanFreelancer9) [inline microsoft-365/compliance/gdpr-arc-Azure.md]:
```suggestion
| ***Human Resource Security (6.4)*** | The customer should determine and assign responsibility for providing relevant training related to protecting personal data. | An overview of the role of Microsoft's Data Protection Officer, the nature of their duties, reporting structure, and contact information.<br>- *Microsoft Cloud Security Policy* [[4](gdpr-arc-Azure.md#4)] see 03. Human Resources Security<br>- *FedRAMP Moderate FedRAMP System Security Plan* [[3](gdpr-arc-Azure.md#3)] see 13.2 "Awareness and Training" (AT) | (39)(1)(b) |
```

File: microsoft-365/compliance/gdpr-arc-Dynamics365.md
Comment (JohanFreelancer9) [inline microsoft-365/compliance/gdpr-arc-Dynamics365.md]:
```suggestion
| ***Organization of Information Security Customer consideration (6.3)*** | The customer should, within their organization, define responsibilities for security and protection of personal data. This may include establishing specific roles to oversee privacy-related matters, including a DPO. Appropriate training and management support should be provided to support these roles. | An overview of the role of Microsoft's Data Protection Officer, the nature of their duties, reporting structure, and contact information.<br>- Microsoft DPO Information [[16](gdpr-arc-Dynamics365.md#16)]<br>- Dynamics 365-ISO27001 Statement of Applicability [[5](gdpr-arc-Dynamics365.md#5)]<br>- Dynamics 365-ISO27018 Statement of Applicability [[6](gdpr-arc-Dynamics365.md#6)]<br>- 6.3 Organization of information security in Compliance Manager [[15](gdpr-arc-Dynamics365.md#15)] | (37)(1)(a), (37)(1)(b), (37)(1)(c), (37)(2), (37)(3), (37)(4), (37)(5), (37)(6), (37)(7), (38)(1), (38)(2), (38)(3), (38)(4), (38)(5), (38)(6), (39)(1)(a), (39)(1)(b), (39)(1)(c), (39)(1)(d), (39)(1)(e), (39)(2) |
```

File: microsoft-365/compliance/gdpr-arc-Office365.md
Comment (JohanFreelancer9) [inline microsoft-365/compliance/gdpr-arc-Office365.md]:
```suggestion
|***Organization of Information Security Customer consideration (6.3)***|The customer should, within their organization, define responsibilities for security and protection of personal data. This may include establishing specific roles to oversee privacy-related matters, including a DPO. Appropriate training and management support should be provided to support these roles.|An overview of the role of Microsoft's Data Protection Officer, the nature of their duties, reporting structure, and contact information.<br>- *Microsoft’s Data Protection Officer* [[18](gdpr-arc-Office365.md#18)]<br>- *Office 365 ISMS Manua*l [[14](gdpr-arc-Office365.md#14)] see *5.3 ORGANIZATIONAL ROLES, RESPONSIBILITIES, AND AUTHORITIES*<br>- 6.3 Organization of information security in [Compliance Manager](https://servicetrust.microsoft.com/ComplianceManager)|(37)(1)(a), (37)(1)(b), (37)(1)(c), (37)(2), (37)(3), (37)(4), (37)(5), (37)(6), (37)(7), (38)(1), (38)(2), (38)(3), (38)(4), (38)(5), (38)(6), (39)(1)(a), (39)(1)(b), (39)(1)(c), (39)(1)(d), (39)(1)(e), (39)(2)|
```

File: microsoft-365/compliance/gdpr-arc-prof-services.md
Comment (JohanFreelancer9) [inline microsoft-365/compliance/gdpr-arc-prof-services.md]:
```suggestion
| ***Human Resource Security (6.4)*** | The customer should determine and assign responsibility for providing relevant training related to protecting personal data. | An overview of the role of Microsoft's Data Protection Officer, the nature of their duties, reporting structure, and contact information.<br>- Microsoft Professional Services ISO/IEC 27001:2013 ISMS Statement of Applicability [[12](gdpr-arc-prof-services.md#12)]<br>- Training and Awareness Program Description [[3](gdpr-arc-prof-services.md#3)] | (39)(1)(b) |
```

File: microsoft-365/compliance/gdpr-dsr-Dynamics365.md
Comment (JohanFreelancer9) [inline microsoft-365/compliance/gdpr-dsr-Dynamics365.md]:
```suggestion
- Properly identify the person and role—such as employee, customer, vendor—by using information that the data subject gave you as part of their request. This information might be a name, an employee ID or customer number, or other identifier.
```

File: microsoft-365/compliance/tagging-and-assessment-in-advanced-ediscovery.md
Comment (JohanFreelancer9) [inline microsoft-365/compliance/tagging-and-assessment-in-advanced-ediscovery.md]:
```suggestion
> In the **Relevance \> Track** tab, expanded issue display, the following viewing options are available: > The recommended next step, such as **Next step: Tagging** can be bypassed (per issue) by clicking the **Modify** button to its right and then selecting a different step in the **Next step**. When the assessment progress indicator has not completed, assessment will be the next recommended option to tag more assessment files and increase statistics accuracy. > You can change the error margin and assess its impact by clicking **Modify** and in the **Assessment level dialog** changing the **Target error margin for recall estimates** and clicking **Update values**. You can also view advanced options in the dialogue box by clicking **Advanced**. > You can view additional assessment level statistics and their impact by clicking **View**. In the displayed Detail results dialog, statistics are available per issue when there are at least 500 tagged assessment files and at least 18 files are tagged as Relevant for the issue. 
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy Review - a few additional changes

**Comment by mypil** [comment]
@shabbysoft  - please merge the commit on the copy/edit review made by JohanFreelancer9 or make the necessary changes on the PR following the suggestions made. Thank you.

**Comment by ghost** [comment]
All done! @mypil 
Thanks for your help @JohanFreelancer9 !

**Comment by mypil** [comment]
@robmazz @markjjo @chrisda @Sirkkuw @LauraWi @denisebmsft @chrfox @MSFTTracyP  - PR has been copy edited and is ready for final review. This PR to correct typo/grammar errors spans across 29 different docs in which you may currently own. I'm tagging all of you and perhaps one of you could possibly review, merge or advise how to proceed with this. 

Thank you.

cc: @kenwith

**Comment by robmazz** [comment]
@mypil - who decided to make this copy edit review and submit these changes? There is nothing wrong about correcting typos and bad grammar, but some of these topics are GDPR-related topics that contain legal statements that, unfortunately, are in the required legal phrasing and not necessarily conforming to our style guide. Are reviewers randomly selecting topics for these types of reviews?

cc: @kenwith

**Comment by mypil** [comment]
@robmazz - This PR is from one of the customers and not from our freelancer pool. The PR copy/edit review done by JohanFreelancer9 is part of our SOP when we manage all PRs within the repo for uniformity and to ensure that it is in accordance to MS style. However, I tagged all the authors of the affected docs so all of you would have the final say on how to proceed with this PR (whether to merge or close it). Hope this clarifies things. I will leave the final decision to you. Thank you.

**Comment by robmazz** [comment]
@mypil  - please back out all the suggested corrections to the topics that start with "gdpr" and resubmit.

**Comment by robmazz** [comment]
@shabbysoft and @mypil - thanks for the efforts with these corrections, but closing this PR due to included content that shouldn't be edited. I will look into blocking these topics so they cannot be edited publicly in the future and so that other topics can still benefit from these types of updates. Thanks!

**Pull request closed without merging**
